FortnitePorting/Export/ExporterInstance.cs

```diff
@@ -51,6 +51,95 @@ public ExporterInstance(EExportTargetType exportType)
         FileExportOptions = AppExportOptions.CreateExportOptions();
     }
 
+    private void ParsePoseAsset(UPoseAsset poseAsset, ExportHeadMeta meta)
+    {
+        /* Only tested when bAdditivePose = true */
+        if (!poseAsset.bAdditivePose)
+        {
+            Log.Warning($"{poseAsset.Name}: bAdditivePose = false is unsupported");
+            return;
+        }
+
+        var poseContainer = poseAsset.PoseContainer;
+        var poses = poseContainer.Poses;
+        if (poses.Length == 0)
+        {
+            Log.Warning($"{poseAsset.Name}: has no poses");
+            return;
+        }
+
+        var poseNames = poseContainer.PoseFNames;
+        if (poseNames is null || poseNames.Length == 0)
+        {
+            Log.Warning($"{poseAsset.Name}: PoseFNames is null or empty");
+            return;
+        }
+
+        /* Assert number of tracks == number of bones for given skeleton */
+        var poseTracks = poseContainer.Tracks;
+        var poseTrackInfluences = poseContainer.TrackPoseInfluenceIndices;
+        if (poseTracks.Length != poseTrackInfluences.Length)
+        {
+            Log.Warning($"{poseAsset.Name}: length of Tracks != length of TrackPoseInfluenceIndices");
+            return;
+        }
+
+        /* Add poses by name first in order they appear */
+        for (var i = 0; i < poses.Length; i++)
+        {
+            var pose = poses[i];
+            var poseName = poseNames[i];
+            var poseData = new PoseData(poseName.PlainText, pose.CurveData);
+            meta.PoseData.Add(poseData);
+        }
+
+        /* Discover connection between bone name and relative location. */
+        for (var i = 0; i < poseTrackInfluences.Length; i++)
+        {
+            var poseTrackInfluence = poseTrackInfluences[i];
+            if (poseTrackInfluence is null) continue;
+
+            var poseTrackName = poseTracks[i];
+            foreach (var influence in poseTrackInfluence.Influences)
+            {
+                var pose = meta.PoseData[influence.PoseIndex];
+                var transform = poses[influence.PoseIndex].LocalSpacePose[influence.BoneTransformIndex];
+                pose.Keys.Add(new PoseKey(
+                    poseTrackName.PlainText, /* Bone name to move */
+                    transform.Translation,
+                    transform.Rotation,
+                    transform.Scale3D,
+                    influence.PoseIndex,
+                    influence.BoneTransformIndex
+                ));
+            }
+        }
+
+        if (poseAsset.RetargetSourceAssetReferencePose is null) return;
+        if (poseAsset.Skeleton is null) return;
+        if (!poseAsset.Skeleton.TryLoad<USkeleton>(out var skeleton)) return;
```

Comment:
does skipping the code below actually cause issues for export? (metahuman skins don't go through this at all)